RunningOrder/Story/Views/StoryList+Logic.swift

```diff
@@ -42,5 +50,40 @@ extension StoryList {
 
             return Color.Identifier.epicColors[epicIndex]
         }
+
+        var allStories: [Story] {
+            var wholeStories: [Story] = []
+            sprintManager.sprints.forEach { sprint in
+                let storiesPerId = storyManager.stories[sprint.id] ?? []
+                wholeStories.append(contentsOf: storiesPerId)
+            }
+
+            return wholeStories
+        }
+
+        /// Returns the stories of a specific sprintId
+        /// - Parameter sprintId: The id of the sprint
+        func stories(for sprintId: Sprint.ID, filter: String? = nil) -> [Story] {
```

Comment:
same as above, It is the manager job to give the logic the stories for the current sprint